<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QWidget" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>636</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>644</width>
    <height>636</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>36</y>
     <width>421</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>421</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>421</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gen">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>36</y>
     <width>133</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>135</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Generate\Refresh List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>132</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Tri Selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>132</y>
     <width>469</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>204</y>
     <width>469</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>204</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Tri Bulles</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>276</y>
     <width>469</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>276</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Tri Insertion</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>348</y>
     <width>469</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>348</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Tri Shell</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>420</y>
     <width>469</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>420</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Tri comptage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>492</y>
     <width>469</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>469</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>492</y>
     <width>85</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>85</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Tri Creation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_all">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>564</y>
     <width>133</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>133</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Test All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clear">
   <property name="geometry">
    <rect>
     <x>156</x>
     <y>564</y>
     <width>133</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>133</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>133</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>CLEAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_selection">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>132</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_bulles">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>204</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_insertion">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>276</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_shell">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>348</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_comptage">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>420</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_creation">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>492</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="log_liste">
   <property name="geometry">
    <rect>
     <x>579</x>
     <y>36</y>
     <width>49</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>49</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QSlider" name="h_slider">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>96</y>
     <width>421</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>3000</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_n">
   <property name="geometry">
    <rect>
     <x>444</x>
     <y>96</y>
     <width>133</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
